### DISCUSSION TITLE
MultiTenancy docs is not up to date esp. Need help.

### DISCUSSION THREAD

**Discussion by KiranAtReconn** [serenity-is/Serenity#7148] [Help]
I am using StartSharp template and implementing MultiTenancy. I am following serene docs and I think there is a little disconnect. Docs seems to be lagging on the info. Can you pls help me address the below highlighted line?
![image](https://github.com/serenity-is/Serenity/assets/149862857/4084c772-ad1b-4e28-8202-888356db2b6c)

**Comment by VictorTomaili** [comment]
inject and use `IPermissionKeyLister`

**Reply by KiranAtReconn** [reply]
awesome! thank you!
